.mergify.yml

```
@@ -13,6 +13,19 @@ pull_request_rules:
             git merge upstream/{{base}}
             git push upstream {{head}}
             ```
+  - name: backport patches to main branch
+    conditions:
+      - merged
+      - label=backport-main
+    actions:
+      backport:
+        assignees:
+          - "{{ author }}"
+        labels:
+          - "backport"
+        branches:
+          - "main"
+        title: "[{{ destination_branch }}](backport #{{ number }}) {{ title }}"
```

This pattern is _slightly_ different from all the other backport actions in this repo (same info, different order for backport # and original PR's title). Fine either way, but this edit uses the existing pattern if that's something we want to preserve:
 ```suggestion
         title: "[{{ destination_branch }}] {{ title }} (backport #{{ number }})"
```